### DISCUSSION TITLE
Is it possible to change the octopus icon to something else?

### DISCUSSION THREAD

**Discussion by walkrideroll** [opentrack/opentrack#1958] [Q&A]
I simply just want to replace the 2D octopus with my own custom 2D image. I see there is a thread called "let's talk about the pose-widget" but after skimming that it looks like it just talks about making it 3D. Would appreciate it if anyone could help!

**Comment by sthalik** [comment]
You can change it but you need to recompile opentrack.

**Reply by walkrideroll** [reply]
> You can change it but you need to recompile opentrack.

Is that easy to do? Is it the "opentrack.exe" or one of the DLLs? Could you tell me what kind of tutorial I could look up for that? Thanks.
